medusa/tv/series.py

```diff
@@ -1223,7 +1223,7 @@ def load_from_indexer(self, tvapi=None):
         self.classification = getattr(indexed_show, 'classification', 'Scripted')
         self.genre = getattr(indexed_show, 'genre', '')
         self.network = getattr(indexed_show, 'network', '')
-        self.runtime = int(getattr(indexed_show, 'runtime', 0))
+        self.runtime = int(getattr(indexed_show, 'runtime', 0) or 0)
```

Comment:
Cant have runtime of 0

In the past used to be try_int with 60 as default.
@p0psicles did you changed that?